tests/integration/test_run_cli.py

```diff
@@ -57,3 +57,108 @@ def test_run_in_shell(pixi: Path, tmp_path: Path) -> None:
         stdout_contains=["a", "a1"],
         env=env,
     )
+
+
+def test_using_prefix_validation(pixi: Path, tmp_path: Path, dummy_channel_1: str) -> None:
+    manifest = tmp_path.joinpath("pixi.toml")
+    toml = f"""
+    [project]
+    name = "test"
+    channels = ["{dummy_channel_1}"]
+    platforms = ["linux-64", "osx-64", "osx-arm64", "win-64"]
+
+    [dependencies]
+    dummy-a = "*"
+    """
+    manifest.write_text(toml)
+
+    # Run the install
+    verify_cli_command(
+        [pixi, "install", "--manifest-path", manifest],
+        ExitCode.SUCCESS,
+    )
+
+    # Validate creation of the pixi file with the hash
+    pixi_file = default_env_path(tmp_path).joinpath("conda-meta").joinpath("pixi")
+    assert pixi_file.exists()
+    assert "environment_lock_file_hash" in pixi_file.read_text()
+
+    # Break environment on purpose
+    dummy_a_meta_files = default_env_path(tmp_path).joinpath("conda-meta").joinpath("dummy-a*.json")
+    import glob
```

Comment:
```suggestion
```

Let's use the glob functionality provided by `pathlib` instead.
In general, I would also prefer top-level imports unless there's a good reason for it